packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -438,6 +438,14 @@ abstract class Widget extends DiagnosticableTree {
 ///  * Use `const` widgets where possible, and provide a `const` constructor for
 ///    the widget so that users of the widget can also do so.
 ///
+///  * When you need to create a reusable piece of UI, prefer using a widget
```

Comment:
nit: Avoid "you" -- see style guide https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-the-passive-voice-recommend-do-not-require

Reply:
Thank you very much for your review! I've addressed your feedback